src/Tasks/BootstrapperUtil/BootstrapperBuilder.cs

```diff
@@ -76,7 +76,7 @@ public class BootstrapperBuilder : IBootstrapperBuilder
         /// </summary>
         public BootstrapperBuilder()
         {
-            _path = Util.DefaultPath;
+            _path = NativeMethodsShared.IsWindows ? Util.DefaultPath : string.Empty;
```

Comment:
why empty over `null`?

Reply:
fear of some code path using _path without doing null checks. the class was marked with `[SupportedOSPlatform("windows")]` so this is no longer an issue.